From 7505b25635d28a5ba41fa74de3749054e89c409f Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 27 Oct 2022 10:36:36 -0700
Subject: [PATCH 03/10] builder: use sudo to call mock

In Clear Linux we use sudo (instead of consolehelper) to let mock have
the right permissions.
---
 builder/kojid | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/builder/kojid b/builder/kojid
index 8b8b1b3..87cb0a6 100755
--- a/builder/kojid
+++ b/builder/kojid
@@ -463,6 +463,8 @@ class BuildRoot(object):
             else:
                 cmd.append('--old-chroot')
         cmd.extend(args)
+        # In Clear Linux we call mock using sudo.
+        cmd.insert(0, "/usr/bin/sudo")
         self.logger.info(format_shell_cmd(cmd))
         workdir = getattr(self, 'workdir', None)
         mocklog = 'mock_output.log'
-- 
2.45.1

